src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -118,6 +120,10 @@ public void testVersion0_97IsValid() throws Exception{
   @Test
   public void testVersion0_98IsValid() throws Exception{
     rootDir = Paths.get(getClass().getClassLoader().getResource("bags/v0_98/bag").toURI());
+    if (System.getProperty("os.name").contains("Windows")){
+    	Path bagitDir = rootDir.resolve(".bagit");
```

Comment:
I will pull code, but leave a comment indicating that these unit tests will fail in Windows, and why, with ref to issue #55